packages/airnode-protocol/contracts/utils/OwnableCallForwarder.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/Address.sol";
+import "./interfaces/IOwnableCallForwarder.sol";
+
+/// @title Contract that forwards the calls that its owner sends
+/// @dev AccessControlRegistry users that want their access control tables
+/// to be transferrable (e.g., a DAO) will use this forwarder instead of
+/// interacting with it directly. There are cases where this transferrability
+/// is not desired, e.g., if the user is an Airnode and is immutably associated
+/// with a single address, in which case the manager will interact with
+/// AccessControlRegistry directly.
+contract OwnableCallForwarder is Ownable, IOwnableCallForwarder {
```

Comment:
the Owner can renounce their ownership, do we want that to happen?

Reply:
Yes, I think this was talked about at the call. Say you want to deploy a `RequestAuthorizerWithManager`, deploy `AirnodeTokenLock` , give it the indefinite whitelister role, then renounce your ownership of the `OwnableCallForwarder` that owns `RequestAuthorizerWithManager`. This allows node operators to use the `RequestAuthorizerWithManager` without having to worry about the manager screwing them over. This is a decent solution to serving chains on which we don't have native decentralized governance (though it only works for authorizers, not beacon/dAPI servers).